modules/Bio/EnsEMBL/Compara/Utils/Test.pm

```diff
@@ -362,6 +362,12 @@ sub get_schema_from_database {
         $schema{$t}->{'COLUMNS'} = $sth->fetchall_hashref('COLUMN_NAME');
         my @pk_columns = $dbh->primary_key_info(undef, undef, $t);
         $schema{$t}->{'PRIMARY_KEY'} = \@pk_columns;
+        $sth = $dbh->statistics_info(undef, $db_name, $t, undef, 'quick');
+        $schema{$t}->{'INDEXES'} = $sth->fetchall_hashref(['INDEX_NAME', 'ORDINAL_POSITION']);
+        foreach my $h (values %{$schema{$t}->{'INDEXES'}}) {
+            delete $_->{TABLE_SCHEM} for values %$h;
+            delete $_->{CARDINALITY} for values %$h;
+        }
```

Comment:
I think it would be better to understand this bit of code (its intention) with a short comment.